dart/test/default_integrations_test.dart

```diff
@@ -92,13 +92,14 @@ void main() {
     expect(true, called);
   }, onPlatform: {'browser': Skip()});
 
-  test('Run zoned guarded calls catches error', () async {
+  test('Run zoned guarded calls catches integrations errors', () async {
     final throwable = StateError('error');
     void callback() {
       throw throwable;
     }
 
     final integration = RunZonedGuardedIntegration(callback);
+
```

Comment:
there are a few code lines with this diff, is it space adding or removing?

Reply:
Mostly formatting i did during the course of the PR.

Reply:
yeah got it, I just could not find out what changed

> @marandaneto I think the failing analyze step was not introduced with this PR:
> 
> <img alt="Bildschirmfoto 2021-01-25 um 16 56 43" width="1250" src="https://user-images.githubusercontent.com/3984453/105730338-889c2180-5f2e-11eb-8f6f-6809b8c7d1ab.png">

I only see formatting stuff, where did you see this?

Reply:
Try running `dart analyze --fatal-infos` in root directory.

Reply:
i have the latest beta, so I don't have this issue

Reply:
yes, works on the beta channel